CardForm/src/main/java/com/braintreepayments/cardform/view/CardEditText.java

```diff
@@ -72,6 +76,36 @@ public CardType getCardType() {
         return mCardType;
     }
 
+    /**
+     * @param masked if {@code true}, the last four digits of the card number will be masked when
```

Comment:
We might want to expand on the JavaDoc to reference `CardNumberTransformation` and/or throw an example here of what masked looks like.

Reply:
It's not the last four that will be masked, but all but the last four.

Reply:
Good catch!